src/index.html

```diff
@@ -11,7 +11,152 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="field">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text"name="surname"
+          required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="name">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          value="12"
+          min="1"
+          max="100" required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+          type="date"
+          name="birthdate"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agreements" required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+
+        <legend>Registration</legend>
+
+
+        <label class="form-field">
```

Comment:
- don't add empty line between elements of different levels of nesting
- don't add more than one empty line
- add empty line only if it helps to improve readability of your code. To my mind, it is redundant here

```suggestion
      <fieldset>
        <legend>Registration</legend>
        <label class="form-field">
```

Reply:
fixxed